### PR TITLE
Adopt zizmor GitHub Actions security scanning tool

### PR DESCRIPTION
**Pull request by blink1073**
Adds support for [zizmor](https://github.com/woodruffw/zizmor), "a tool for finding security issues in GitHub Actions CI/CD setups."
It is still in development, but already useful.  Missing features are a [GitHub Action](https://github.com/woodruffw/zizmor/issues/69), and a non-zero [exit code](https://github.com/woodruffw/zizmor/issues/86) so we can fail builds.  For now this applies the recommended fixes and sets it up as a security scan source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,11 +255,44 @@ By default, all files in the S3 directory are uploaded. When the `dry_run` input
 is set to anything other than `false`, no files are uploaded, but instead the
 filename along with the resulting location in the bucket is printed.
 
+## Create Release Branch
```
Comment (alcaeus) [inline README.md:258]: Wasn't this already added by #59?

**Review by alcaeus — COMMENTED** [review]
Great idea to add checks for GitHub Actions. Just one question about the create-branch content.

File: README.md (hunk 1 above)
Comment (blink1073) [reply README.md:258]: Drat, forgot to create a new branch, reverting.

Comment (blink1073) [reply README.md]: Okay, pulling from main resolved it.

**Review by alcaeus — APPROVED** [review]

**Pull request merged**